auth/openid_connect.adoc

```diff
@@ -211,3 +216,74 @@ with enough administrative privileges to update it:
 * ssh to the appliance as root
 * Run _appliance_console_cli --extauth-opts_ *local_login_disabled=false*
 
+
+[[oidc-configuration-cli]]
+== Configuring OIDC using the CLI
+
+Another way to configure OpenID-Connect on the appliance is to use the appliance
+console CLI instead of the above steps. The `--oidc-config` subcommand of
+the  appliance console CLI allows one to configure the appliance for OpenID-Connect
+Authentication by updating both Apache configurations as well as the necessary
+Administrative UI settings. The `--oidc-unconfig` subcommand is also provided
+for unconfiguring the appliance OpenID-Connect Authentication and reverting
+the appliance to Database Authentication.
+
+Usage of the new Appliance Console CLI subcommands are as follows:
+
+=== Configure OpenID-Connect
+
+To configure the appliance for OpenID-Connect Authentication:
+
+```
+# appliance_console_cli --oidc-config
+                        [--oidc-client-host]
+                        --oidc-url
+                        --oidc-client-id
+                        --oidc-client-secret
+                        [--oidc-enable-sso]
+```
+
+When the `--oidc-client-host` is not specified, the configured appliance host
+is used for the ServerName and OpenID-Connect Redirect URI in the
+OpenID-Connect Apache configuration file. The `--oidc-client-host` is
+usually the externally reachable FQDN of the appliance.
+
+Configuring OpenID-Connect requires the following parameters:
+
+[options="header",cols="<2,<5"]
+|====================================================================
+|Parameter            | Description
+|--oidc-url           | This is the OpenID-Connect Provider URL
+|--oidc-client-id     | The OpenID-Connect Provider Cliend ID
+|--oidc-client-secret | The OpenID-Connect Provider Client Secret
+|====================================================================
+
+The `--oidc-url` varies for different providers. An example provider
+URL for keycloak:
+
+```
+--oidc-url=http://keycloak-server:8080/auth/realms/miq-realm/.well-known/openid-configuration
+```
+With *miq-realm* being the Keycloak Realm this OpenID-Connect
+appliance client is being configured for.
+
+The `--oidc-client-id` is the Client ID specified for
+the appliance in the OpenID-Connect provider, usually
+just `<miq-appliance-fqdn{gt}`
```

Comment:
`{gt}` ?

Reply:
yea, depending on where a > is sometimes rendered as >;  I will update all the ones that render properly as >.